rbac/server/api/main.py

```diff
@@ -62,6 +63,45 @@
     )
     app.config.HTTP_SESSION = aiohttp.ClientSession(connector=conn, loop=loop)
 
+    await asyncio.sleep(5)
+
+    LOGGER.warning("Creating default admin user and role.")
+    async with aiohttp.ClientSession(connector=conn, loop=loop) as session:
+        LOGGER.info("Creating Next Admin user...")
+        admin_user = {
+            "name": getenv("NEXT_ADMIN_NAME"),
+            "username": getenv("NEXT_ADMIN_USER"),
+            "password": getenv("NEXT_ADMIN_PASS"),
+            "email": getenv("NEXT_ADMIN_EMAIL"),
+        }
+        user_response = await session.post(
+            "http://rbac-server:8000/api/users", json=admin_user
+        )
+        user_response_json = await user_response.json()
+        user_next_id = user_response_json["data"]["user"]["id"]
+        LOGGER.info("Creating NextAdmin role...")
+        admin_role = {
+            "name": "NextAdmins",
+            "owners": user_next_id,
+            "administrators": user_next_id,
+        }
+        role_response = await session.post(
+            "http://rbac-server:8000/api/roles", json=admin_role
+        )
+        role_response_json = await role_response.json()
+        role_next_id = role_response_json["data"]["id"]
+        LOGGER.info("Adding Next Admin to NextAdmins role...")
+        add_user = {
+            "pack_id": None,
+            "id": user_next_id,
+            "reason": None,
+            "metadata": None,
+        }
+        await session.post(
+            ("http://rbac-server:8000/api/roles/%s/members" % role_next_id),
```

Comment:
Standard when not in a Logger statement is typically .format(role_next_id) with {} where it needs to be